Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -213,6 +215,50 @@ public final class CoreDataManager: StorageManagerType {
             return debugMessages
         }
     }
+
+    /// Note that we have to enumerate all the files in our sandbox to get these properties, so this takes some time.
+    /// It is intended _only_ for use when logging a crash, or other high-value log which won't get in the user's way.
+    private static func storageSizeLogProperties() -> [String: Int64]? {
+        let directoryUrls = [
+            "Bundle": Bundle.main.bundleURL,
+            "Documents": FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!,
+            "Library": FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask).first!,
+            "Temp": FileManager.default.temporaryDirectory,
+        ]
```

Comment:
I know we're using this to log _just before_ crashing the app but I would still be on the safe side to avoid unintended consequences.

```suggestion
    private static func storageSizeLogProperties() -> [String: Int64]? {
        let directoryUrls: [String: URL] = [
            "Bundle": Bundle.main.bundleURL,
            "Documents": FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first,
            "Library": FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask).first,
            "Temp": FileManager.default.temporaryDirectory,
        ].compactMapValues { $0 }
```

Reply:
Thanks, I meant to come back to improve this one and forgot. This is a nice clean fix.